src/cf-tables.less

```diff
@@ -436,14 +459,14 @@
 
     .table__entry-header-on-small {
         & > tbody td:first-child {
+            padding-bottom: .75em;
+            margin-bottom: unit( 10px / @base-font-size-px, em);
             margin-top: 0;
+            border-bottom: 1px solid @table-border;
```

Comment:
Border should come before margin, as we follow the box model from inside out (width/height > padding > border > margin).

Also, while fixing that, let's order multiple rules of the same type (`margin-bottom` and `margin-top` in this case) in the same order as the shorthand rule would (top, left, bottom, right)